netdimm/netdimm.py

```diff
@@ -231,6 +231,11 @@ def reboot(self) -> None:
             # set time limit to 10 minutes.
             self.__set_time_limit(10)
 
+    def set_time_limit(self) -> None:
```

Comment:
I would recommend taking the time limit in here, instead of hardcoding it to 10 minutes. I know it'll only be passed in from elsewhere, but at least that makes this interface generic for anyone.